mu4e/mu4e-server.el

```diff
@@ -643,7 +643,7 @@ PATH must be below the root-maildir."
 Optionally to MAILDIR and optionally setting FLAGS. If MAILDIR is
 nil, message will be moved within the same maildir.
 
-At least one of MAILDIR and FLAGS must be specified. Note that
+At least one of MAILDIR or FLAGS must be specified. Note that
 even when MAILDIR is nil, this is still a filesystem move, since
```

Comment:
I think the "and" is fine here (same below), so let's leave that it is

Reply:
Okay. I amended the commit.